src/sandstorm/grain.capnp

```diff
@@ -633,7 +633,20 @@ interface SessionContext {
   #   saveLabel: A string petname to give this label. This will be displayed to the user as the name
   #          for this capability.
   #
-  # (eg. window.parent.postMessage({powerboxRequest: {rpcId: myRpcId, query: [{}]}}, "*")
+  # e.g.:
+  #   window.parent.postMessage({
+  #     powerboxRequest: {
+  #       rpcId: myRpcId,
+  #       query: [
+  #         {
+  #           tags: [
+  #             { id: "10145426045091321204" },
```

Comment:
Yes, I agree that it should be a valid node-capnp value, but the issue is that there are multiple things which node-capnp will accept, but when doing an implicit capnproto-struct-to-JS-object, there can be only one which it will emit in the thing that it generates to `postMessage()` to the client.  Whatever that is should be the canonical JSON encoding of a capnproto 64-bit int.

Since node-capnp is not an option in frontend javascript code (which will be the immediate consumer of this postMessage() that we send back to the app with the sturdyref and the descriptor at issue), we can't assume that apps will have that available to do the reverse conversion.  So I think we ought to specify what that encoding will be, so that apps can rely on *something* without having to pull in node-capnp on the backend.  And today, for better or worse, that canonical encoding, as implemented by the capnproto JSON encoder/decoder, is a decimal string.  So what we `postMessage()` back to apps should likewise be a decimal string.  At which point for simplicity on the app side, we should probably just use decimal strings in the requests, even if we also patch the frontend to accept hex strings.  This way, if app authors treat capnp ids as opaque identifiers, they can rely on string equality for equality testing.

I'll make the frontend able to accept either the decimal or hex representation of just the frontendref-specific interfaces, but after thinking about all this, I intend to continue using the decimal representation in the documentation and my example app, since that's more canonically the JSON representation of a capnproto 64-bit int.